service/history/transferQueueStandbyTaskExecutor.go

```diff
@@ -166,24 +166,12 @@ func (t *transferQueueStandbyTaskExecutor) processWorkflowTask(
 
 		executionInfo := mutableState.GetExecutionInfo()
 
-		var taskQueue *taskqueuepb.TaskQueue
-		var taskScheduleToStartTimeoutSeconds = int64(0)
-		if mutableState.GetExecutionInfo().TaskQueue != transferTask.TaskQueue {
-			// this workflowTask is an sticky workflowTask
-			// there shall already be an timer set
-			taskQueue = &taskqueuepb.TaskQueue{
-				Name: transferTask.TaskQueue,
-				Kind: enumspb.TASK_QUEUE_KIND_STICKY,
-			}
-			taskScheduleToStartTimeoutSeconds = int64(timestamp.DurationValue(executionInfo.StickyScheduleToStartTimeout).Seconds())
-		} else {
-			taskQueue = &taskqueuepb.TaskQueue{
-				Name: transferTask.TaskQueue,
-				Kind: enumspb.TASK_QUEUE_KIND_NORMAL,
-			}
-			workflowRunTimeout := timestamp.DurationValue(executionInfo.WorkflowRunTimeout)
-			taskScheduleToStartTimeoutSeconds = int64(workflowRunTimeout.Round(time.Second).Seconds())
+		taskQueue := &taskqueuepb.TaskQueue{
+			Name: transferTask.TaskQueue,
+			Kind: enumspb.TASK_QUEUE_KIND_NORMAL,
 		}
+		workflowRunTimeout := timestamp.DurationValue(executionInfo.WorkflowRunTimeout)
+		taskScheduleToStartTimeoutSeconds := int64(workflowRunTimeout.Round(time.Second).Seconds())
```

Comment:
this logic will create a mismatch, e.g. sticky task is transmuted into a normal task, but corresponding schedule to start timer still exits

Reply:
that should not matter. when the timer fires, if the task is started, its a no-op; if the task has not started, it just re-create it and clear stickiness, no special logic needed.

Reply:
no this PR itself is special handling sticky queue staleness.

anyway, plz put a "experimental" mark here, we need to eventually move the sticky queue management to matching service (source of truth)